stage2/protos/stivale.c

```diff
@@ -1,53 +1,22 @@
-#include <stdint.h>
-#include <stddef.h>
-#include <stdbool.h>
-#include <protos/stivale.h>
-#include <lib/elf.h>
-#include <lib/blib.h>
+#include <drivers/pic.h>
```

Comment:
Let's not shuffle includes around, alright? :)

Reply:
Oups my IDE sorted them automatically :/